src/module.c

```diff
@@ -13605,7 +13624,7 @@ int RM_SetRem(RedisModuleKey *key, RedisModuleString **eles, size_t numeles, siz
 
 /* Return 1 as member of the key or 0 as not member of the key. */
 int RM_SetIsMember(RedisModuleKey *key, RedisModuleString *ele) {
-    if (!key->value || key->value->type != OBJ_SET) return 0;
+    if (!key || !key->value || key->value->type != OBJ_SET) return 0;
```

Comment:
What about setting errno in these is-member cases?

I suggest returning -1 on errors here, so users cab distinguish between zero (not a member) and error.

!key->value is not an error, but !key and wrong type are errors.

Reply:
!key for empty key if open key with reading mode. !key-value for empty key if open key with writing mode.

Reply:
Yes, looks good now. :+1: